src/containers/person/PersonProfile.js

```diff
@@ -47,9 +47,15 @@ const initialFetch = (id, query) => {
 }
 
 const PersonProfile = compose(
-  prefetch(({ dispatch, params: { id }, query }) =>
+  prefetch(({ store, dispatch, params: { id }, query }) =>
   Promise.all([
-    dispatch(fetchPerson(id)),
+    dispatch(fetchPerson(id))
+    .then(({ payload: {shared_communities} }) => {
```

Comment:
also, try loading a profile with an invalid user id -- you may need to short-circuit return here if action.error is set and only get shared_communities afterward